projects/plugins/jetpack/extensions/blocks/payment-buttons/edit.js

```diff
@@ -105,6 +110,8 @@ function PaymentButtonsEdit( { clientId } ) {
 					</div>
 				</Placeholder>
 			) }
+			{ /* Explicitly position the wpcom upgrade nudge wrapper */ }
+			<div className="jetpack-block-upgrade-banner-container"></div>
```

Comment:
The wrapper is added as the first children of the block container dynamically which should result in this same markup, so you shouldn't need to explicitly add this div here. Doesn't that work?

Reply:
Yeah good catch, I think this is left over from an earlier, discarded attempt. I should make this change.